sky/shell/platform/mojo/main_mojo.cc

```diff
@@ -38,10 +39,12 @@ class MojoApp : public mojo::ApplicationDelegate,
     mojo::icu::Initialize(app);
     tracing_.Initialize(app);
 
-    Shell::Settings settings;
+    blink::SkySettings settings;
+    settings.enable_observatory = true;
```

Comment:
This should probably be a port number, with 0 = observatory disabled, rather than a bool. But that can wait for another time.